### PR TITLE
Assessment config loading

### PR DESCRIPTION
**Pull request by nategbrown9**

### CODE REVIEW

File: bridge-client/src/commonMain/kotlin/org/sagebionetworks/bridge/kmm/shared/models/AssessmentInfo.kt [hunk 1]
```diff
@@ -32,30 +34,37 @@ import kotlinx.serialization.SerialName
...
     /* AssessmentInfo */
     @SerialName("type")
     val type: String? = null
-)
+) {
+
+    /** Path portion of config url */
+    @Transient
+    val configPath: String
+    get() = configUrl.let {Url(configUrl).encodedPath}
```
Comment (syoung-smallwisdom) [inline bridge-client/src/commonMain/kotlin/org/sagebionetworks/bridge/kmm/shared/models/AssessmentInfo.kt]: Is this auto-generated?

**Review by syoung-smallwisdom — APPROVED** [review]
looks good

File: bridge-client/src/commonMain/kotlin/org/sagebionetworks/bridge/kmm/shared/models/AssessmentInfo.kt (hunk 1 above)
Comment (nategbrown9) [reply bridge-client/src/commonMain/kotlin/org/sagebionetworks/bridge/kmm/shared/models/AssessmentInfo.kt]: No, I added it.

Comment (syoung-smallwisdom) [inline bridge-client/src/commonMain/kotlin/org/sagebionetworks/bridge/kmm/shared/models/AssessmentInfo.kt]:
```suggestion
        get() = Url(configUrl).encodedPath
```

configUrl isn't null so it doesn't need the `let`, correct? Also, should be indented.

Comment (nategbrown9) [reply bridge-client/src/commonMain/kotlin/org/sagebionetworks/bridge/kmm/shared/models/AssessmentInfo.kt]: Oops, didn't remove the let after confirming with Alx that configUrl is non-nullable

**Review by syoung-smallwisdom — APPROVED** [review]
looks good

**Pull request merged**